script/install.sh

```diff
@@ -763,19 +763,25 @@ install_dub() {
         log "no dub binaries available for $OS"
         return
     fi
-    local url=http://code.dlang.org/download/LATEST
-    logV "Determining latest dub version ($url)."
-    dub="dub-$(fetch $url)"
+    local latestMirrors=(
+        "http://code.dlang.org/download/LATEST"
+        "http://dlang.github.io/dub/LATEST"
```

Comment:
Why not https?

Reply:
I think simply because it wasn't using https before